cirq-core/cirq/devices/grid_qubit.py

```diff
@@ -218,8 +218,8 @@ def from_diagram(diagram: str, dimension: int) -> List['GridQid']:
         AAAAAAA
         ```
 
-        You can use any character other than a hyphen to mark a qid. As an
-        example, the qids for the Bristlecone device could be represented by
+        You can use any character other than a hyphen, period or space to mark a
+        qid. As an example, the qids for a Sycamore device could be represented by
```

Comment:
The grid below is a [Bristlecone grid](https://github.com/quantumlib/Cirq/blob/5ad8e6e5c26ed0618c81731b0ec09e84435bf61f/cirq-google/cirq_google/devices/known_devices.py#L249). We can replace this with the Sycamore grid from the same file if you'd like to use Sycamore specifically as the example - otherwise this should be changed back to Bristlecone.